### PR TITLE
[new release] eqaf (0.6)

### PR DESCRIPTION
**Pull request by dinosaure**
Constant-time equal function on string

- Project page: <a href="https://github.com/mirage/eqaf">https://github.com/mirage/eqaf</a>
- Documentation: <a href="https://mirage.github.io/eqaf/">https://mirage.github.io/eqaf/</a>

##### CHANGES:

- remove build dependency on dune (@CraigFe, mirage/eqaf#16)
- add bigarray-compat and optional dependencies (@hannesm, mirage/eqaf#17)
- add `select_int`, `one_if_not_zero`, `zero_if_not_zero` (@cfcs, @dinosaure, mirage/eqaf#19, mirage/eqaf#18)

### CODE REVIEW

**Comment by dinosaure** [comment]
Tests fails due to the required `nasm` binary - otherwise, it seems that `check/check.exe` returns a good result.

File: packages/eqaf/eqaf.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+depends: [
+  "ocaml"          {>= "4.03.0"}
+  "dune"
+  "alcotest"       {with-test}
+  "crowbar"        {with-test}
+]
+
+depopts: [
+  "cstruct" {>= "4.0.0"}
```
Comment (kit-ty-kate) [inline packages/eqaf/eqaf.0.6/opam]: Constraints should not be in the `depopts` field. Here it means "i'll treat this package as a dependency if `cstruct >= 4.0.0` is installed (*usually* not what you want), but if you remove the constraints and add a conflict field like this `conflicts: ["cstruct" {< "4.0.0"}]` it means "cstruct is an optional dependency and if it is installed, this packages won't work with cstruct < 4.0.0"

**Comment by kit-ty-kate** [comment]
Could you return the fix upstream?

**Comment by dinosaure** [comment]
Done into `master`, thanks!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**